chains/dodotestnet/metadata.yaml

```diff
@@ -0,0 +1,27 @@
+blockExplorers:
+  - apiUrl: https://testnet-scan.dodochain.com/api
+    family: blockscout
+    name: DODOchain Testnet (Sepolia) Explorer
+    url: https://testnet-scan.dodochain.com
+blocks:
+  confirmations: 0
+  reorgPeriod: 0
+  estimateBlockTime: 13
+chainId: 53457
+deployer:
+  name: DODO
+  url: http://dodoex.io
+displayName: DODOChain Testnet
```

Comment:
Optional but since this name is a big long, a `displayNameShort: DODOChain` may be a good addition